components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/dao/ApiMgtDAO.java

```diff
@@ -9383,6 +9425,37 @@ public void deleteKeyManagerConfigurationById(String id, String organization) th
 
     }
 
+    public KeyManagerPermissionConfigurationDTO getKeyManagerPermissions(String keyManagerUUID) throws APIManagementException {
```

Comment:
Seems the line length is exceeding 120. Let's format it.

Reply:
Fixed in 1d58204afd0d2b23f33e003063649b697cf2e32a